tests/common/dualtor/dual_tor_utils.py

```diff
@@ -1347,7 +1347,7 @@ def increase_linkmgrd_probe_interval(duthosts, tbinfo):
     duthosts.shell_cmds(cmds=cmds)
 
 @pytest.fixture(scope='module')
-def dualtor_ports(request, duthosts, rand_one_dut_hostname, enum_frontend_asic_index):
+def dualtor_ports(request, duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_frontend_asic_index):
```

Comment:
Can you please explain how this change will affect and need to do this ? Did we test this change to be working fine on pizza boxes ?

Reply:
![image](https://user-images.githubusercontent.com/46945843/188726128-45709c94-79c8-4a1d-890d-8b184ed13dbb.png)

Reply:
Did you add your HWSKU information in this file ?